### PR TITLE
Update and (almost) completing Dutch strings for 1.5

### PR DESCRIPTION
**Pull request by ghost**
Only privacy policy is not translated, but is included. Hopefully I have time to translate this another time (you can set your own privacy policy now anyway).

### CODE REVIEW

**Review by Gargron — APPROVED** [review]

File: config/locales/activerecord.nl.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+en:
```
Comment (ykzts) [inline config/locales/activerecord.nl.yml]: `nl:`?

File: config/locales/nl.yml [hunk 2]
```diff
@@ -112,7 +261,7 @@ nl:
     bad_password_msg: Goed geprobeerd hackers! Ongeldig wachtwoord
     confirm_password: Voer jouw huidige wachtwoord in om jouw identiteit te bevestigen
     description_html: Hierdoor worden alle gegevens van jouw account <strong>permanent, onomkeerbaar</strong> verwijderd en wordt deze gedeactiveerd. Om toekomstige identiteitsdiefstal te voorkomen, kan op deze server jouw gebruikersnaam niet meer gebruikt worden.
-    proceed: Account verwijderen
+    proceed: Account verwijderen
```
Comment (ykzts) [inline config/locales/nl.yml]: remove trailing space.

File: config/locales/nl.yml [hunk 3]
```diff
@@ -124,7 +273,9 @@ nl:
       content: Veiligheidsverificatie mislukt. Blokkeer je toevallig cookies?
       title: Veiligheidsverificatie mislukt
     '429': Te veel verbindingsaanvragen.
-    noscript: Schakel JavaScript in om Mastodon te kunnen gebruiken.
+    noscript: Schakel JavaScript in om de webapplicatie van Mastodon te gebruiken. Als alternatief kan je een Mastodon-app zoeken voor jouw platform.
+   exports:
```
Comment (ykzts) [inline config/locales/nl.yml]: remove this line.

File: config/locales/nl.yml [hunk 4]
```diff
@@ -124,7 +273,9 @@ nl:
...
+   exports:
```
Comment (ykzts) [inline config/locales/nl.yml]: remove this line.

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by Gargron** [comment]
Good idea to run `i18n-tasks normalize`

**Comment by jeroenpraat** [comment]
@Gargron I understand that `i18n-tasks normalize` is only possible when having a (non-docker) development instance set up. I only have one instance running using Docker. I think it's a bit overkill to set up a dev-instance for translations.

**Review by ykzts — APPROVED** [review]

**Pull request merged**